apps/full-site-editing/full-site-editing-plugin/common/index.js

```diff
@@ -56,4 +56,13 @@ const BlockPatternsMoved = () => {
 	);
 };
 
-registerPlugin( 'block-patterns-moved', { render: BlockPatternsMoved } );
+// Atomic sites without Gutenberg enabled don't need to this plugin.
+// Check that gutenberg is enabled.
+if (
+	// for Atomic (this doesn't work... need a different check)
+	getPlugin( 'gutenberg' ) ||
```

Comment:
I believe these are editor (Gutenberg) plugins. Not WordPress plugins. Confusing 🙂 

We might check for the presence of `__experimentalLibrary` export on [block-editor](https://unpkg.com/browse/@wordpress/block-editor@3.10.0/build-module/components/index.js). That's [what's being used](https://github.com/WordPress/gutenberg/blob/4830c2c2b95dc612b73efffbd3ed03cd93cd83fa/packages/edit-post/src/components/layout/index.js#L19) to render the patterns right now.